### ISSUE TITLE
Script erroring out

### ISSUE THREAD

**Issue by ryanjpearson** [tomtor/HDL-deflate#2]
Hi, I'm new to Python, so forgive me if I'm not seeing something basic.  When I try to run the script, this is the output.  I have python 3.9 and myHDL 0.11.0 installed.  Please advise.  Thanks!

C:\HDL-deflate-master>py deflate.py
IBSIZE 512
Traceback (most recent call last):
  File "C:\HDL-deflate-master\deflate.py", line 1670, in <module>
    d.convert(initial_values=False)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\_block.py", line 342, in convert
    return converter(self)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_toVerilog.py", line 177, in __call__
    genlist = _analyzeGens(arglist, h.absnames)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 170, in _analyzeGens
    v.visit(tree)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\ast.py", line 407, in visit
    return visitor(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 1070, in visit_Module
    self.generic_visit(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\ast.py", line 415, in generic_visit
    self.visit(item)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\ast.py", line 407, in visit
    return visitor(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 1158, in visit_FunctionDef
    self.visit(n)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\ast.py", line 407, in visit
    return visitor(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 521, in visit_Assign
    self.visit(target)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\ast.py", line 407, in visit
    return visitor(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 466, in visit_Attribute
    self.setAttr(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 482, in setAttr
    self.visit(node.value)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\ast.py", line 407, in visit
    return visitor(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 962, in visit_Subscript
    self.accessIndex(node)
  File "C:\Users\rpearson\AppData\Local\Programs\Python\Python39\lib\site-packages\myhdl\conversion\_analyze.py", line 986, in accessIndex
    self.visit(node.slice.value)
AttributeError: 'Name' object has no attribute 'value'

**Comment by tomtor** [comment]
I did just test with MyHDL 0.10 and 0.11 and those work fine, but I am using Python 3.6.9, the Ubuntu 18.04 default.

I don't think the Python version is the real issue, but it would help to check if you retry with Python 3.6. 
The issue is most likely caused by other non default packages. How did you install MyHdl?

I use `pip install myhdl`

**Comment by ryanjpearson** [comment]
I should have mentioned I'm running windows 10.  I had trouble running pip at first, so I first installed myhdl by getting the 0.9.0 package here per myhdl.orgs instructions.  https://sourceforge.net/projects/myhdl/files/myhdl/
py setup.py install

With 0.9.0 there was a problem with "async", and I figured out how to run pip, so I installed 0.11.0 as follows:

py -m pip install --upgrade myhdl.

I want to remake deflate v, because I want a 32768 OBSIZE and to remove the compression part of it to save resources.  I thought about editing deflate.v directly, but I wanted to be sure I wasn't missing anything.  Thanks!

**Comment by tomtor** [comment]
Hmm, I'm afraid running `pip install upgrade` after having manually installed myhdl first will break your install.

I would suggest wiping all Python installed stuff and restarting with a fresh `pip install`, that should fix this issue.

**Comment by ryanjpearson** [comment]
My procedure for wiping Python stuff:
I ran: py -m pip uninstall myhdl
I ran Windows add/remove programs to remove the python installation
I deleted \AppData\Local\Programs\Python and \AppData\Local\pip

I did a fresh install with Python 3.9 and myhdl 0.11.0.  The same problem occurred.

I wiped and then did a fresh install with Python 3.6.8 (there was no windows installer executable offered for 3.6.9).  The script completed and produced the deflate.v file.

I will now change the adjustable parameters and run the script again.  Thanks!
